<launch>
    <node
        pkg = "gnss_l86_interface"
        name = "gnss_l86_interface_node"
        type = "gnss_l86_interface_node"
        args = "/dev/serial0"
        required = "true"
    />

    <node 
        pkg = "gy_88_interface"
        name = "gy_88_interface_node"
        type = "gy_88_interface_node"
        args = "300"
        required = "true"
    />

    <!-- <node
        pkg = "cartesian_pose"
        type = "cartesian_pose_node"
        name = "cartesian_pose_node"
    /> -->
</launch>
